### PR TITLE
update french translations

### PR DESCRIPTION
**Pull request by NouaaTW**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/french.txt [hunk 1]
```diff
@@ -1361,52 +1362,52 @@ Skip Tutorial
...
-== 
+== Permet de rendre les cartes avec plus de détails
 
 Renderer
-== 
+== Moteur de rendu graphique
 
 default
-== 
+== défault
```
Comment (Chairn) [inline data/languages/french.txt:1377]: défaut

File: data/languages/french.txt [hunk 2]
```diff
@@ -1361,52 +1362,52 @@ Skip Tutorial
...
 
-custom
-== 
+Custom
```
Comment (Chairn) [inline data/languages/french.txt:1379]: custom, don't modify headlines

File: data/languages/french.txt [hunk 3]
```diff
@@ -1361,52 +1362,52 @@ Skip Tutorial
...
+== Personnalisé
 
 Graphics cards
-== 
+== Carte graphique
 
-auto
-== 
+Auto
```
Comment (Chairn) [inline data/languages/french.txt:1385]: same here, auto

File: data/languages/french.txt [hunk 4]
```diff
@@ -1361,52 +1362,52 @@ Skip Tutorial
...
-== 
+== Impossible de trouver un serveur tutoriel
 
 Super
-== 
+== Super
 
 Team %d
-== 
+== Equipe
```
Comment (Chairn) [inline data/languages/french.txt]: Équipe %d

**Comment by NouaaTW** [comment]
well, lots of mistakes. This is the first time i've touched this file, I'll try to do better next time.. @Chairn thx for fixing

**Pull request merged**